andela/AndelaEatsUI
# #160536338 Implement menus listing view

#### What does this PR do?
- Adds page for admin to view all menus

#### Description of Task to be completed?
- Add menus component and route for menus page
- Add test for menus component

#### How should this be manually tested?
- Login to the app
- Navigate to the admin dashboard from the user drop-down
- Click on menus on the side bar to switch to the menus tab
- Ensure to see a table listing all menus

#### What are the relevant pivotal tracker stories?
- [#160536338](https://www.pivotaltracker.com/story/show/160536338)

#### Background Context
This is UI only feature

#### Screenshots (If Applicable)
<img width="1440" alt="screen shot 2018-09-27 at 10 19 18 am" src="https://user-images.githubusercontent.com/32123313/46136141-ce515e00-c23e-11e8-8acc-40539f08a2e3.png">
